vsintegration/src/FSharp.Editor/FSharpKeywords.fs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+module Microsoft.VisualStudio.FSharp.Editor.FSharpKeywords
+  
+let keywordsWithDescription : (string * string) list =
+    [ "abstract",  """Indicates a method that either has no implementation in the type in which it is declared or that is virtual and has a default implementation."""
```

Comment:
Yes, move ``keywordsWithDescription`` into ``lexhelp.fs``.  It's OK if there is some dupliction as long a it's all in the same file.  But each string should go into ``FSComp.txt``, without a number but with a unique identifier.  Search for a similar string to see how to look up the localized strings.